connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -777,6 +783,30 @@ private void startTask(ConnectorTaskId taskId) {
         worker.startTask(taskId, taskConfig, connConfig, this, initialState);
     }
 
+    private Callable<Void> getTaskCallable(final ConnectorTaskId taskId, final TaskStatus.State action) {
+        return new Callable<Void>() {
+            @Override
+            public Void call() throws Exception {
+                switch (action) {
+                    case UNASSIGNED:
```

Comment:
Well, if you want to match the use of `DESTROYED`, `RUNNING` probably makes the most sense since that is the target state you want the connector/task to be in. But I'm not picky, either one works.